src/transformState.js

```diff
@@ -5,7 +5,27 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (const item in actions) {
```

Comment:
```suggestion
  for (const action of actions) {
```
actions - це масив об'єктів. Для масивів використовується цикл for of.
ТОді ти можеш напряму перевіряти item.type всередині